### PR TITLE
Format code with cargo-fmt

### PR DESCRIPTION
**Pull request by cyphersnake**
This PR includes only automatic formatting changes related to cargo-fmt with default settings.

### CODE REVIEW

File: src/glv.rs [hunk 1]
```diff
@@ -25,17 +25,23 @@ pub fn decompose_slow(scalar: &G1ScalarField) -> (G1ScalarField, G1ScalarField)
...
+    (
+        G1ScalarField::from(quotient),
+        G1ScalarField::from(remainder),
+    )
 }
 
 const LMDA1: u128 = 0xac45a4010001a402; // lambda high 64 bit
 const LMDA0: u128 = 0x00000000ffffffff; // lambda low 64 bit
-// const INV2: u128 = 0x1;                // 2**256 // lambda - 128th bit
+                                        // const INV2: u128 = 0x1;                // 2**256 // lambda - 128th bit
```
Comment (drouyang) [inline src/glv.rs]: Seems this line needs to be reverted manually

**Review by drouyang — APPROVED** [review]
How can we enable auto cargo-fmt check automatically?

File: src/glv.rs (hunk 1 above)
Comment (Gigatron) [reply src/glv.rs]: probably just remove it since it's commented out code

Comment (drouyang) [reply src/glv.rs]: Sure.

For the record, INV2 is the highest bit of INV, which was not used in the calculation below and thus commented out.

**Pull request merged**